SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -82,15 +89,38 @@ Worker_RequestId USpatialWorkerConnection::SendCreateEntityRequest(TArray<FWorke
 	{
 		Data.Emplace(SpatialGDK::OwningComponentDataPtr(Component.schema_type), Component.component_id);
 	}
+
 	return Coordinator->SendCreateEntityRequest(MoveTemp(Data), Id, TOptional<uint32>(), SpanId);
 }
 
+Worker_RequestId USpatialWorkerConnection::SendCreateEntityRequest(TArray<FWorkerComponentData> Components, const Worker_EntityId* EntityId,
```

Comment:
I'd say for most command requests you probably never want to call it without a retry data object so it might be worth only having one overload.

The exception to that is for a entity command requests where it's probably fairly common to send one without any intention of retrying. Although there would could still just have one overload unless we are worried about the performance of having the check the retry data.

Reply:
sure, can remove the non-retry overload